GitLab/scripts/cleanup.sh

```diff
@@ -1,26 +1,28 @@
 #!/bin/bash
 
+set -o errexit -o nounset -o pipefail
```

Comment:
I would keep the short syntax.
It is consistent with the rest of the repository. And there is really no reason for the long version.
Same goes for all other usages of `set -eu`

Reply:
I try to follow a set of best practices regarding shell scripting, inspired by such articles:

* https://kvz.io/bash-best-practices.html
> Use long options (logger --priority vs logger -p). If you're on cli, abbreviations make sense for efficiency. but when you're writing reusable scripts a few extra keystrokes will pay off in readability and avoid ventures into man pages in the future by you or your collaborators.

* https://www.franzoni.eu/shell-scripting-when-to-use-the-long-or-the-shor/
> But whenever you write a script that should, later on, be used and read by other people - or just yourself in the future - the long form is often self-explanatory.

Of course if you really want to, I'll switch to short options

Reply:
I personally find those options in the `if the option is very common` category from the second link.
If you are familiar with them you would recognise them in both short and long format. If you are not, then even if you see the long format, you most probably need to check the manpage.
So I vote for the short syntax here.

Reply:
As you wish. I have just made the change.